app/models/gift.rb

```diff
@@ -22,14 +22,16 @@ def has_name_or_pseud
     end
   end
 
-  validates_uniqueness_of :pseud_id, scope: [:work_id], message: ts("You can't give a gift to the same person twice.")
+  validates_uniqueness_of :pseud_id,
+    scope: :work_id,
```

Comment:
Just trying to keep the lines a little shorter to make them more readable on my screen. work_id doesn't need to be in an array, but it's not really an important change.